nerfstudio/models/splatfacto.py

```diff
@@ -732,7 +732,10 @@ def get_outputs(self, camera: Cameras) -> Dict[str, Union[torch.Tensor, List]]:
             tile_bounds,
         )  # type: ignore
         if (self.radii).sum() == 0:
-            return {"rgb": background.repeat(int(camera.height.item()), int(camera.width.item()), 1)}
+            return {
+                "rgb": background.repeat(int(camera.height.item()), int(camera.width.item()), 1),
+                "depth": torch.ones(int(camera.height.item()), int(camera.width.item()), 1) * 10,
```

Comment:
should we include `"accumulation"` also?

Reply:
True. This line also needs to be edited:

https://github.com/nerfstudio-project/nerfstudio/blob/17873aa19551316bf9b8ee38f12a50d36a1ac2df/nerfstudio/models/splatfacto.py#L670